### ISSUE TITLE
How to add options in editor

### ISSUE THREAD

**Issue by excelant100** [lukasalexanderweber/stitching#37]
I have no idea how to add the options from "-h" on console to a code editor such as Pycharm. How can I achieve this?

**Comment by lukasalexanderweber** [comment]
You can't. You need to use the Stitcher Object in your Script as shown in the readme. You can pass all the Parameter in the constructor of the Stitcher object. See e.g. here:

https://github.com/lukasalexanderweber/stitching_tutorial/blob/master/docs/Stitching%20Tutorial.md#affine-stitcher

```
settings = {
            "crop": False,
            "confidence_threshold": 0.5}    

stitcher = Stitcher(**settings)
panorama = stitcher.stitch(imgs)
```

**Comment by excelant100** [comment]
Amazing, I finally managed to make it work. 

By the way, do you know which settings to play with are the best to will help optimize stitching time? I want to do complete it as fast as possible.

Thank you so much!

**Comment by lukasalexanderweber** [comment]
The defaults are not that bad, you could work with `final_megapix` if you don't need the very best resolution of the result. I don't know your use case but maybe you can skip seam estimation or exposure compensation.

**Comment by lukasalexanderweber** [comment]
closed with 821dbe43fabb7a5a02d8f98570ac4b00280648d9
